lib/fuel-auth.js

```diff
@@ -136,10 +153,15 @@ module.exports = class FuelAuth {
 				if (body.refreshToken) {
 					this.refreshToken = body.refreshToken;
 				}
-
-				this.accessToken = body.accessToken || null;
-				this.expiration = body.expiresIn ? process.hrtime()[0] + body.expiresIn : null;
-
+				
+				if(this.apiVersion === 2){
+					this.accessToken = body.access_token || null;
+					body.accessToken = this.accessToken;
```

Comment:
i don't see why we need this line?

Reply:
With authv2 the accessToken was renamed to access_token. So the body.accessToken does not exists in that case and is assigned the value of body.access_token.
I think this exists to support both v1 and v2. If v1 is to be deprecated this could probably be refactored.

The body needs this value, because it is resolved in line 165